app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/WindowWidget.java

```diff
@@ -1663,7 +1662,7 @@ public void onContextMenu(GeckoSession session, int screenX, int screenY, Contex
         }
         mContextMenu.show(REQUEST_FOCUS);
 
-        mWidgetPlacement.tintColor = 0x555555FF;
+        mWidgetPlacement.tintColor = Windows.GREY;
```

Comment:
nit: The color is `GRAY`, the little gray men from space are `GREYS` (Mostly a joke, I think the US is they only place that spells it `gray`).

Reply:
Updated